bin/fun-init.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/* eslint-disable quotes */
+
+'use strict';
+
+const program = require('commander');
+
+const examples =
+  `
+  Examples:
+
+    $ fun init
+    $ fun init foo/bar
+    $ fun init gh:foo/bar
+    $ fun init gl:foo/bar
+    $ fun init bb:foo/bar
+    $ fun init github:foo/bar
+    $ fun init gitlab:foo/bar
+    $ fun init bitbucket:foo/bar
+    $ fun init git+ssh://git@github.com/foo/bar.git
+    $ fun init hg+ssh://hg@bitbucket.org/bar/foo
+    $ fun init git@github.com/foo/bar.git
+    $ fun init hg@bitbucket.org/bar/foo
+    $ fun init https://github.com/foo/bar.git
+    $ fun init /path/foo/bar
+    $ fun init -n fun-app -V foo=bar /path/foo/bar
+  `;
+
+const parseVars = (val, vars) => {
+  /*
+   * Key-value pairs, separated by equal signs
+   * keys can only begin with letters, dollar signs, and underscores, other characters can only contain letters, numbers, and underscores
+   * values can be any character
+   */
+  const group = val.match(/(^[a-zA-Z$_][a-zA-Z\d_]*)=(.*)/);
+  vars = vars || {};
+  if (group) {
+    vars[group[1]] = group[2];
+  }
+  return vars;
+};
+
+program
+  .name('init')
+  .usage('[options] [location]')
+  .description('Initializes a new fun project')
+  .option('-o, --output-dir [outputDir]', 'where to output the initialized app into', '.')
+  .option('-n, --name [name]', 'name of your project to be generated as a folder', 'fun-app')
+  .option('-i, --no-input [noInput]', 'disable prompting and accept default values defined template config', false)
```

Comment:
```suggestion
  .option('-i, --no-input', 'disable prompting and accept default values defined template config')
```
Options starting with --no prefix negate the boolean value of the following word. In this case, `--no-input` sets the value of `program.input` to false, and the value is true if the option is not specified.

Reply:
你说得对，我是按照这个逻辑处理的

Reply:
'[noInput]' and the default value 'false' are not necessary in this case. When running with `--no-input`，the value of `program.input` is `false`, when running without `--no-input`, the value is `true`.

Reply:
-i 短名是否可以去掉，感觉和它的含义 input 是相反的

Reply:
> '[noInput]' and the default value 'false' may be not necessary in this case. When running with `--no-input`，the value of `program.input` is `false`, when running without `--no-input`, the value is `true`.

OK